tests/rest/client/test_sync.py

```diff
@@ -426,8 +444,9 @@ def test_read_receipt_with_empty_body(self):
             "POST",
             "/rooms/%s/receipt/m.read/%s" % (self.room_id, res["event_id"]),
             access_token=self.tok2,
+            custom_headers=[("User Agent", user_agent)],
```

Comment:
the header has a hyphen-minus in the name
```suggestion
            custom_headers=[("User-Agent", user_agent)],
```

Reply:
Is that standard header formatting or is it specific to this application? I tried sifting through the uses of custom_headers and make_request in the code but could only find a couple of instances.

Reply:
It's part of the HTTP standard. I'm not sure if it's strictly case sensitive, but at the very least the hyphen-minus is required and spaces probably aren't even allowed :)

Reply:
I see. I'll have to read about that then. Thanks!